packages/queries/src/types/ColumnValue.ts

```diff
@@ -21,4 +27,13 @@ export type ColumnValue<
 type ColumnType<
     TColumnDef,
     TColumn extends keyof TColumnDef,
-> = TColumnDef[TColumn] extends { type: infer TType } ? TType : never;
+> = TColumnDef[TColumn] extends
+    | { type: infer TNullableType; nullable: true }
+    | undefined
```

Comment:
why `| undefined`? can you explain this case?

Reply:
Oh that looks like a bug in the types.

Reply:
Can you fix it? Should be a really easy fix.

Reply:
Ahh.. okay. On it